packages/cli/src/scripts/create.js

```diff
@@ -1,16 +1,26 @@
 import stdout from '../utils/stdout';
 import ControllerFor from '../models/network/ControllerFor';
+import promptPassword from '../utils/promptPassword'
 
 export default async function createProxy({ packageName, contractAlias, initMethod, initArgs, network, txParams = {}, force = false, networkFile = undefined }) {
   if (!contractAlias) throw Error('A contract alias must be provided to create a new proxy.')
   
   const controller = ControllerFor(network, txParams, networkFile)
   if (controller.isLib) throw Error('Cannot create a proxy for a library project')
-  
   await controller.checkContractDeployed(packageName, contractAlias, !force);
-  const proxy = await controller.createProxy(packageName, contractAlias, initMethod, initArgs);
-
+  let proxy
+  try {
+  proxy = await controller.createProxy(packageName, contractAlias, initMethod, initArgs);
+  } catch(e) {
+    if (e.message === "signer account is locked"){
+      await promptPassword(txParams.from ? txParams.from : web3.eth.accounts[0])
```

Comment:
@jcarpanelli I'm unsure what is the best approach here, as errors are not propagated up to the command due to the `try ... finally` block in the scripts, what are your thoughts?

Reply:
One possible approach (not saying it's the best one, it's just one approach) would be wrapping the script call in the `<command>.js` in a `try/catch`statement. In practical terms, it would be the same as wrapping code in the `script.js` file, because eventually the script will throw the error (and maybe isolating the try/catch to an external function would be great too, to avoid making the code dirty).